ibm_secrets_manager_sdk/version.py

```diff
@@ -17,4 +17,4 @@
 """
 Version of ibm_secrets_manager_sdk
 """
-__version__ = '2.1.5'
+__version__ = '2.1.6'
```

Comment:
This change should be removed. Publication automation bumps version automatically.